src/core/dfn-finder.js

```diff
@@ -97,9 +97,11 @@ function generateMethodNamesWithArgs(operationName, argsAst) {
   operationNames.push(requiredOperation);
   const optionalOps = optional.map((_, index) => {
     const optionalArgs = optional.slice(0, index + 1).join(", ");
-    const result = `${operationName}(${requiredArgs}${
-      optionalArgs ? `, ${optionalArgs}` : ""
-    })`;
+    const args =
+      requiredArgs && optionalArgs
+        ? `${requiredArgs}, ${optionalArgs}`
+        : requiredArgs || optionalArgs || "";
```

Comment:
:fire: :100:

Reply:
Can this be `[...required, ...optional.slice(0, index + 1)].join(", ")`?

Reply:
It sure can :)